backend/tests/test_auth_api.py

```
@@ -1,161 +1,70 @@
+from app.models import User
+from app.utils import auth_utils  
+
+client = TestClient(app)
+def test_password_hashing():
+    raw_password = "test123"
+    hashed = auth_utils.hash_password(raw_password)
+    assert auth_utils.verify_password(raw_password, hashed)
+
+
+
+def create_test_user(db: Session, email: str, password: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_test_user`

(ANN201)